compiler+runtime/src/cpp/jank/runtime/detail/object_util.cpp

```diff
@@ -30,6 +35,17 @@ namespace jank::runtime::detail
       o);
   }
 
+  bool equal(char const lhs, object_ptr const rhs)
+  {
+    if(!rhs || rhs->type != object_type::character)
+    {
+      return false;
+    }
+
+    auto const typed_rhs = expect_object<obj::character>(rhs);
+    return typed_rhs->to_hash() == static_cast<unsigned int>(lhs);
+  }
+
   bool equal(object_ptr const lhs, object_ptr const rhs)
```

Comment:
You're likely just following the usage of this `bool`, rather than `native_bool`, but that one should be changed, too. My code doesn't get reviewed though, so sometimes these things slip though. :grin: